### PR TITLE
Feature/rh 2 homepage

### PR DESCRIPTION
**Pull request by fawadmehmood1**
# Description

Please include a summary of the changes and the related issue.

- Completed homepage

## Type of change

Please select type of issue addressed,

- [x] New feature ( RH-2 ) 

# Dependences / Libraries Installed

Please list the packages and libraries installed (if any)

- [x] ( @gsap/react) 
- [x] (react-fast-marquee)

# How Has This Been Tested By The Developer?

- Verified if the ui is according to figma

# Checklist:

- [x] I have performed a self review of my code.

- [x] I have removed consoles, debuggers and unnecessary code blocks.

- [x] My code follows the style guidelines of this project

- [x] My changes generate no new warnings or errors

- [ ] I have commented my code, particularly in hard-to-understand areas

- [ ] New and existing unit tests pass locally with my changes

- [ ] Any dependent changes have been merged and published in downstream modules

- [x] I understand that I am accountable for any code added in this PR, and any negative impact will effect my performance evaluation

### CODE REVIEW

File: app/(landing-page)/page.tsx [hunk 1]
```diff
@@ -1,7 +1,302 @@
...
+            ease: "none",
+          },
+          ">-0.5",
+        );
+    }
+  });
+
+  return (
+    <div className="min-h-screen">
+      {/* section# 1*/}
```
Comment (imrankhan-stixor) [inline app/(landing-page)/page.tsx:94]: remove this

File: app/(landing-page)/page.tsx [hunk 2]
```diff
@@ -1,7 +1,302 @@
...
+          />
+
+          <div className="flex h-full">
+            <div ref={image1Ref} className=" shrink-0 overflow-hidden">
+              <Image
+                src={Building}
+                width={1920}
+                height={1600}
+                alt=""
+                className="mx-auto transition-transform duration-500 group-hover:translate-y-[-55%] group-hover:scale-[1.15]"
```
Comment (imrankhan-stixor) [inline app/(landing-page)/page.tsx:114]: use -translate for negative movement

File: app/(landing-page)/page.tsx [hunk 3]
```diff
@@ -1,7 +1,302 @@
...
+            <div
+              ref={image2Ref}
+              className=" shrink-0 -translate-x-[200%]  overflow-hidden"
+            >
+              <Image
+                src={Strassenansicht}
+                width={1920}
+                height={1600}
+                alt=""
+                className="mx-auto transition-transform duration-500 group-hover:translate-y-[-55%] group-hover:scale-[1.15]"
```
Comment (imrankhan-stixor) [inline app/(landing-page)/page.tsx:127]: same here

File: app/(landing-page)/page.tsx [hunk 4]
```diff
@@ -1,7 +1,302 @@
...
+                Leben <br /> Zwischen <br /> stadtpark <br />
+                <span className=" font-semibold text-primary">& </span>
+                alster
+              </p>
+            </div>
+          </div>
+        </div>
+      </section>
+
+      {/* section# 2 */}
```
Comment (imrankhan-stixor) [inline app/(landing-page)/page.tsx:144]: remove

File: app/(landing-page)/page.tsx [hunk 5]
```diff
@@ -1,7 +1,302 @@
...
+            Lifestyle vereint. Diese exklusive Anlage mit nur 9 Einheiten bietet
+            eine gelungene Mischung aus Ruhe und urbaner Eleganz, typisch für
+            das Flair von Winterhude. Ein wahrer Schatz, der sich nahtlos in die
+            Umgebung einfügt und dennoch seinen eigenen unverwechselbaren
+            Charakter besitzt.
+          </p>
+        </div>
+      </section>
+
+      {/* section# 3 */}
```
Comment (imrankhan-stixor) [inline app/(landing-page)/page.tsx:180]: remove

File: app/(landing-page)/page.tsx [hunk 6]
```diff
@@ -1,7 +1,302 @@
...
+      <section className="relative">
+        <section className="relative z-10 bg-bone bg-opacity-50">
```
Comment (imrankhan-stixor) [inline app/(landing-page)/page.tsx:182]: why are two relative used here, remove one if not needed

File: app/(landing-page)/page.tsx [hunk 7]
```diff
@@ -1,7 +1,302 @@
...
+          <div className="container flex flex-col justify-between gap-6 pb-[6.263rem] pt-[2.738rem] lg:flex-row lg:gap-[10vw]">
+            <div className="mt-[3.563rem] lg:order-1 lg:w-[50%] lg:max-w-[47.5rem] 2xl:w-full">
```
Comment (imrankhan-stixor) [inline app/(landing-page)/page.tsx:184]: use w-3/6 instead of w-[50%]

File: app/(landing-page)/page.tsx [hunk 8]
```diff
@@ -1,7 +1,302 @@
...
+              <h2 className="text-[clamp(1.6rem,2.4vw,2.813rem)] text-primary sm:w-[50%] lg:w-[70%] 2xl:w-full 2xl:leading-[3.813rem]">
```
Comment (imrankhan-stixor) [inline app/(landing-page)/page.tsx:185]: here as well

File: app/(landing-page)/page.tsx [hunk 9]
```diff
@@ -1,7 +1,302 @@
...
+                gerecht zu werden. Hier vereinen sich großzügige Räume mit
+                durchdachtem Design.
+              </p>
+            </div>
+
+            <div className="lg:order-0 flex max-w-[29.688rem] flex-col justify-between gap-5 uppercase text-primary sm:flex-row lg:w-[30%] lg:flex-col lg:gap-0 2xl:w-full">
+              <div>
+                <div className="mb-3.5 flex w-full items-end gap-5 lg:gap-2">
+                  <span className="town-houses-numbers">4</span>
+                  <span className="town-houses-tag">STADTHÄUSER</span>
```
Comment (imrankhan-stixor) [inline app/(landing-page)/page.tsx:203]: what are these two classes doing? can't seem to find them in tailwind

File: app/(landing-page)/page.tsx [hunk 10]
```diff
@@ -1,7 +1,302 @@
...
+              <div className="town-houses-tag self-end">
+                <p>1 - 4,5 ZIMMER</p>
+
+                <p>115 - 168 M2</p>
+              </div>
+            </div>
+          </div>
+        </section>
+
+        {/* section# 4 */}
```
Comment (imrankhan-stixor) [inline app/(landing-page)/page.tsx:226]: remove

File: app/(landing-page)/page.tsx [hunk 11]
```diff
@@ -1,7 +1,302 @@
...
+          <div className="grid w-full max-w-[47.5rem] flex-1 grid-cols-[53%_21%_26.4%] grid-rows-[45.9%_18.3%_24.3%] place-content-end ">
+            <div className="col-span-2 row-start-1 row-end-2">
+              <Image src={Building} alt="" />
+            </div>
+            <div className="col-span-2 col-start-2 row-span-2">
+              <Image src={Bildergruppe} alt="" className="h-full" />
+            </div>
+          </div>
+          <div className="flex-1 self-end">
+            <Image src={InnenraumStaffel} width={760} height={494} alt="" />
```
Comment (imrankhan-stixor) [inline app/(landing-page)/page.tsx:237]: add alt message

File: app/(landing-page)/page.tsx [hunk 12]
```diff
@@ -1,7 +1,302 @@
...
+              </div>
+            </div>
+          </div>
+        </section>
+
+        {/* section# 4 */}
+        <section className="container my-[6.25rem] flex flex-col gap-16 md:flex-row md:gap-10 2xl:mt-3.5">
+          <div className="grid w-full max-w-[47.5rem] flex-1 grid-cols-[53%_21%_26.4%] grid-rows-[45.9%_18.3%_24.3%] place-content-end ">
+            <div className="col-span-2 row-start-1 row-end-2">
+              <Image src={Building} alt="" />
```
Comment (imrankhan-stixor) [inline app/(landing-page)/page.tsx:230]: add alt message

File: app/(landing-page)/page.tsx [hunk 13]
```diff
@@ -1,7 +1,302 @@
...
+            </div>
+            <div className="col-span-2 col-start-2 row-span-2">
+              <Image src={Bildergruppe} alt="" className="h-full" />
```
Comment (imrankhan-stixor) [inline app/(landing-page)/page.tsx:233]: add alt message

File: app/(landing-page)/page.tsx [hunk 14]
```diff
@@ -1,7 +1,302 @@
...
+
+        <Image
+          src={Pattern}
+          alt=""
+          fill
+          className="absolute !-left-5 !bottom-2.5 !top-auto hidden max-h-[58.375rem] max-w-[21.25rem] object-cover lg:block"
+        />
+      </section>
+
+      {/* section# 5 */}
```
Comment (imrankhan-stixor) [inline app/(landing-page)/page.tsx:249]: remove

File: app/(landing-page)/page.tsx [hunk 15]
```diff
@@ -1,7 +1,302 @@
...
+            </div>
+          </div>
+          <div className="flex-1 self-end">
+            <Image src={InnenraumStaffel} width={760} height={494} alt="" />
+          </div>
+        </section>
+
+        <Image
+          src={Pattern}
+          alt=""
```
Comment (imrankhan-stixor) [inline app/(landing-page)/page.tsx:243]: add alt message

File: app/(landing-page)/page.tsx [hunk 16]
```diff
@@ -1,7 +1,302 @@
...
+                <li key={marqueItem} className="marquee-list-item">
+                  <span className="marquee-item flex-1">{marqueItem}</span>
+                </li>
+              ))}
+            </ul>
+          </Marquee>
+        </div>
+      </section>
+
+      {/* section# 6 */}
```
Comment (imrankhan-stixor) [inline app/(landing-page)/page.tsx:289]: remove

**Review by imrankhan-stixor — CHANGES_REQUESTED** [review]

File: app/(landing-page)/page.tsx (hunk 1 above)
Comment (fawadmehmood1) [reply app/(landing-page)/page.tsx:94]: @imrankhan-stixor , The comments are there to help us easily identify which section of code corresponds to which part of the layout.

File: app/(landing-page)/page.tsx (hunk 6 above)
Comment (fawadmehmood1) [reply app/(landing-page)/page.tsx:196]: they both have different purpose

File: app/(landing-page)/page.tsx (hunk 9 above)
Comment (fawadmehmood1) [reply app/(landing-page)/page.tsx:217]: same styling is applied to multiple elements, that's why these utility classes are here.

**Review by imrankhan-stixor — APPROVED** [review]

**Pull request merged**